src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -135,6 +136,9 @@ public virtual bool TryCreate(FunctionMetadata functionMetadata, out FunctionDes
                 case BindingType.ManualTrigger:
                     triggerParameter = ParseManualTrigger(triggerMetadata);
                     break;
+                case BindingType.ApiHubTrigger:
+                    triggerParameter = ParseApiHubTrigger((ApiHubBindingMetadata)triggerMetadata, typeof(Stream));
```

Comment:
If this is taking a Stream, work may be required in the NodeFunctionInvoker to handle conversions from Stream to objects that can be passed into Node code. The invoker will already handle Stream->string though.